src/server/cea_608_encoder/scene_utility.py

```diff
@@ -1,14 +1,37 @@
 import src.server.cea_608_encoder.caption_string_utility as utils
 
-def create_bytes_for_scene_background_color(color: str):
-    pass
+background_colors = {"white": 0x0, "green": 0x2, "blue": 0x4, "cyan": 0x6, 
+    "red": 0x8, "yellow": 0xa, "magenta": 0xc, "black": 0xe}
 
+def create_bytes_for_scene_background_color(color: str, transparency = False):
+    byte_list = []
+    first_byte = 0x00
+    second_byte = 0x00
+
+    color = color.lower()
```

Comment:
if some evil person wanted to pass in an integer instead of a string for the color name what would happen here

Reply:
Added the check